grapher/controls/ShareMenu.tsx

```diff
@@ -231,6 +233,7 @@ class EmbedMenu extends React.Component<{
                     onFocus={(evt) => evt.currentTarget.select()}
                     value={`<iframe src="${url}" loading="lazy" style="width: 100%; height: 600px; border: 0px none;"></iframe>`}
                 />
+                {this.manager.embedDialogAdditionalElements}
```

Comment:
I'm not sure how I feel about this. This is the same pattern I had added before, under the reasoning I wanted to keep Grapher and Explorer loosely coupled, but it's a bit hacky.

At this point I think it's likely that Grapher and Explorer merge, so maybe we move the hide controls component itself here?

I'm doing an attempt at fixing the hackiness in our Grapher params once and for all on my branch right now, and maybe after that it will be clear what the best thing to do is.

Reply:
Great! I agree, it does feel hacky to just pass this around to be displayed here, but I didn't really see a better way.
I'm certainly interested what you could come up with in your PR, and will wait with this PR until yours is in.